modulefiles/build_hera_intel.lua

```diff
@@ -27,6 +27,8 @@ load("srw_common")
 load(pathJoin("nccmp", os.getenv("nccmp_ver") or "1.8.9.0"))
 load(pathJoin("nco", os.getenv("nco_ver") or "4.9.3"))
 
+load(pathJoin("wgrib2", os.getenv("wgrib2_ver") or "2.0.8"))
+
```

Comment:
Since wgrib2 is loaded in modulefiles/srw_common.lua, this entry can be removed.

Reply:
Removed.